lib/logstash-logger/formatter.rb

```diff
@@ -36,6 +36,14 @@ def build_event(message, severity, time)
 
       event['host'] ||= HOST
 
+      LogStashLogger.config.custom_fields.each do |field, msg|
+        if msg.respond_to? (:call)
+          event[field] = msg.call()
+        elsif msg.is_a? String
```

Comment:
The explicit check for `String` type probably isn't necessary. I can imagine wanting to pass in a numeric value, or a custom object. As long as the custom value can be converted to JSON, `LogStash::Event` should be happy.